src/DynamoCoreWpf/UI/Themes/Modern/DynamoModern.xaml

```diff
@@ -2498,7 +2498,7 @@
         </Setter>
     </Style>
 
-    <Style x:Key="ContextMenuStyle" TargetType="{x:Type ContextMenu}">
+    <Style x:Key="NodeContextMenuStyle" TargetType="{x:Type ContextMenu}">
```

Comment:
Is this rename necessarily since this seems to be generic context menu style? At least you are applying on connector instead of node

Reply:
@QilongTang I took this from Ollie's code. Not sure what the best practice is there but his contextmenu changes haven't been merged to `master` yet, so I just took this Style and dropped it where his is. That way I can just borrow from this style rather than creating a new one. Thoughts?

Reply:
Yeah, I am in supportive of reusing this, with that reason I feel we dont have to rename then..

Reply:
@M-JULIANI I thought we reverted the changes here but we did not which brought those regressions because package manager UI is depending on `ContextMenuStyle`